### DISCUSSION TITLE
Überwachung PV Anlage

### DISCUSSION THREAD

**Discussion by Gamehunter1989** [arendst/Tasmota#19673] [Support request] [answered]
Hallo zusammen,

die Entwicklung der Tasmota ist beeindruckend.

Ich habe leider zu spät von der Umsetzung gelesen. Somit hat meine PV Anlage schon einige kW/h an Strom erzeugt. Ich möchte gern im Zähler Energie Gesamt den ist Wert anpassen. 

Wie mache ich das?

Lg Mark

**Comment by HofeBY** [comment]
Garnicht.
Die Zählerwerte eines Eichgültigen Stromzählers lassen sich aus guten Gründen nicht nach Gusto des Betreiber tunen.

**Comment by Noschvie** [comment]
@Gamehunter1989 was möchtest du genau ändern: Die Gesamtsumme im Tasmota GUI, oder? Mach mal einen Screenshot

**Comment by Gamehunter1989** [comment]
![image](https://github.com/arendst/Tasmota/assets/59651641/9fe8c956-625d-4f74-bb26-d3c25adbb94f)
@Noschvie genau

Engergie gesamt    282,85 kWh

Diesen Wert möchte ich ändern. 

Da es ja kein geeichter Zähler ist, sollte es, hoffe ich, machbar sein.

**Reply by Noschvie** [reply]
Sollte mit EnergyTotal1 möglich sein.

https://tasmota.github.io/docs/Commands/#power-monitoring

EnergyTotal<x>
Get or set Energy Total values, parameters:
<x> = meter number (default is 1)
<value> = set new value in Wh, 0 for reset

**Reply by Gamehunter1989** [reply]
Das habe ich mal probiert, wohl aber falsch gemacht. Geht das über die Interne Konsole des Tasmota?

**Reply by Gamehunter1989** [reply, marked as answer]
Okay, jetzt habe ich es. Mann muss den Zeitstempel mit einfügen. Will man einen klaren Wert, so muss man auch den energytoday reseten.

Dieser Befehl ist somit die Lösung: energytotal 2330000 2023-10-06T00:00:00

Danke für eure Unterstützung und Zeit.
